circle.yml

```diff
@@ -0,0 +1,85 @@
+version: 2
+
+workflows:
+  version: 2
+  default:
+    jobs:
+      - build
+      - release
+jobs:
+  build:
+    working_directory: ~/code
```

Comment:
Is this line required?

Reply:
It was captured from our implementation in [events repo](https://github.com/mapbox/mapbox-events-android/blob/74c1e851277530cb4531db861486fc030cd484a6/.circleci/config.yml#L11). I don't know whether there are any benefits to this approach.